worker/grpc_requests.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	pb "github.com/koraygocmen/gravitational/jobscheduler"
+	"google.golang.org/grpc"
+)
+
+// registerWorker dials the scheduler GRPC server and registers
+// the calling worker with the worker's GRPC server address.
+// Worker's GRPC server address is later used by the scheduler to dial
+// worker to start/stop/query jobs.
+func registerWorker() {
+	conn, err := grpc.Dial(config.Scheduler.Addr, grpc.WithInsecure())
```

Comment:
TODO remove insecure when using TLS?

Reply:
Exactly, will change to TLS when basic functionality is done. Adding TODO now